1.9/networking/service-and-pod-networking.md

```diff
@@ -0,0 +1,422 @@
+---
+post_title: Service and Pod Networking
+menu_order: 0.5
+---
+
+This document describes how to specify networking modes and assign ports in the service definition for a DC/OS Service you are creating.
+
+# VIPs
+
+We recommend using [virtual addresses (VIPs)](/docs/1.9/usage/service-discovery/virtual-ip-addresses/) to make ports management easier. VIPs map traffic from a single virtual address to multiple IP addresses and ports. They simplify inter-app communication and implement a reliable service-oriented architecture.
+
+# Networking Modes
+
+DC/OS services and pods declare `networks` the same way. Three modes of networking are supported:
+
+## `host` Networking
+
+In `host` networking, an application shares the network namespace of the Mesos agent process, typically the host network namespace.
+
+## `container` Networking
+
+In container networking, an application should be allocated its own network namespace and IP address;
+Mesos network isolators are responsible for providing backend support for this.
+When using the Docker containerizer, this translates to a Docker "user" network. You can name your container network in the `networks.container` parameter of your service definition. If you do not provide a name for your container network, it will have the default container network name, `DEFAULT NAME`.
+
+## `container/bridge` Networking
+
+Similar to `container` networking, an application should be allocated its own network namespace and IP address;
+Mesos CNI provides a special `mesos-bridge` that application containers are attached to.
+When using the Docker containerizer, this translates to the Docker "default bridge" network.
+
+Bridge mode networking allows you to map host ports to ports inside your containers. It is particularly useful if you are using a container image with fixed port assignments that you can't modify.
+
+**Note:**: The UCR and Docker Containerizer support all network modes.
+
+# Usage
+
+* A service can join one or more `container` mode networks. When joining multiple container networks, additional restrictions are imposed on *port-mapping* entries (see *Port Mappings* for details).
+* An application can only join one `host` mode network. `host` networking is the default if your service definition does not declare a `networks` field.
+* An application can only join one `container/bridge` network.
+* An application cannot mix networking modes. You must specify a single `host` network, a single `container/bridge` network, or one or more `container` networks.
+
+# Ports for Services
+
+DC/OS services declare ports differently than pods. The following section is only relevant to services.
```

Comment:
replace these two sentences with: `This section describes how to declare ports for services.`